executor/common_linux.h

```diff
@@ -128,11 +129,9 @@ struct nlmsg {
 	char* pos;
 	int nesting;
 	struct nlattr* nested[8];
-	char buf[1024];
+	char buf[4096];
```

Comment:
This increase could probably also be a separate commit with an explanation that it's a preparatory commit for WiFi fuzzing.

Reply:
Created a separate commit.